br-webui/views/network.liquid

```diff
@@ -181,21 +181,85 @@ function isValidIP(ip) {
 }
 
 function setCompanionEthernetIP() {
-	var ip_addr = prompt("Please enter a new IP Address");
+	var value = document.getElementById("current-config").innerText;
+	console.log(value);
+	var mode = value.trim();
+	console.log(mode);
+	if (mode === "Manual") {
+		var ip_addr = prompt("Please enter a new IP Address in the 192.168.2.x subnet");
+
+		var valid_ip = isValidIP(ip_addr)
+		if (!valid_ip) {
+			alert("Invalid IP address: " + ip_addr);
+			return
+		}
+	
+		if (confirm("Are you sure you want to change the ethernet IP to " + ip_addr + "?")) {
+			socket.emit('set default ip', ip_addr);
+		}
+		alert("New IP will take affect after reboot");
+	}
+	else {
```

Comment:
fix the style here.

Reply:
I am not sure what style you are talking about.

Reply:
https://github.com/bluerobotics/internal/blob/master/software/guidelines/style.md

Reply:
Use one true brace style per astyle doc.